### DISCUSSION TITLE
Saleor Auth SDK and Supabase Auth

### DISCUSSION THREAD

**Discussion by magick93** [saleor/saleor#14459] [General]
Hello

I'm looking at using Supabase as both the underlying postgres database and the auth provider. 

Supabase uses postges and [gotrue](https://github.com/supabase/gotrue) for user auth.

I have a few general questions about how to integrate Supabase Auth.

1. Does the Saleor Auth SDK only work with OIDC? Supabase Auth can integrate with OIDC but is itself not an OIDC IDP.
2. Would it be possible to re-use the Supabase JWTs in Saleor? Currently Supabase Auth only allows signing JWT with HS256 but there are plans to support [RS256](https://github.com/orgs/supabase/discussions/12759). If this is not possible, can you recommend a better solution to integrate with Suapbase auth?

Thanks

**Comment by jha-r** [comment]
@magick93  did you figure out, I need help with this, appreciate your guidance.

**Reply by magick93** [reply]
No, I decided to not continue with Saleor.
